### ISSUE TITLE
Build Issues with using this package on Win Systems

### ISSUE THREAD

**Issue by adityaras** [devongovett/unplugin-parcel-macros#12]
Our Project utilizes `Webpack`; and we have added the following `macros.webpack()` to the list of plugins in webpack config.

On **Mac** platforms everything runs perfectly without any issues but on **Win**  based systems we get a `Bad character escape sequence error` seemingly due to the styles being loaded in the index file which have a `\u` in their paths which Win seems to be considering as escape sequences!

**Comment by adityaras** [comment]
SS of Error
(With some paths Blurred)

![image](https://github.com/user-attachments/assets/1cd599c0-242d-48b8-941b-75461b023661)

**Comment by adityaras** [comment]
A potential fix could be simply removing the filename Path in the hash calculation; this avoids issue with how these paths are calculated on different systems

**Comment by adityaras** [comment]
https://github.com/devongovett/unplugin-parcel-macros/pull/13

**Comment by devongovett** [comment]
This appears to fix the issue for now, but eventually we might need to find a way to add the path back. For example, if you use `style({backgroundImage: 'url(../foo.png)'})`, the image should be resolved relative to the file that `style` is called in. However without the file path it will be resolved relative to the root of the project. I tried to fix this but was unsuccessful - looks like it will require changes in unplugin itself - so I bailed for now.
